readme.txt

```diff
@@ -133,6 +133,14 @@ http://www.youtube.com/watch?v=dJrykKQGDcs
 
 == Changelog ==
 
+= 4.6.22 [January 09, 2025] =
+
+* Fixed
+
+  * Fixed the read property on null warnings for WP v6.7
```

Comment:
Write in a way like Fixed PHP warnings( no need to highlight the issue) for WP v6.7